index.html

```diff
@@ -0,0 +1,152 @@
+            <div class="welcome__content">
+                <label class="welcome__text--blue welcome__text--big-in-sm">Welcome</label>
+                <label class="welcome__text--big">A Great Place to Receive Care</label>
+                <label class="welcome__text--muted">Get your best looking smile now!</label>
+                <div class="welcome__buttons">
+                    <button class="button button--blue">Join Us</button>
+                    <button class="button button--border">Learn More</button>
+                </div>
+            </div>
+            <div class="welcome__empty"></div>
+        </div>
+    </div>
+    <div class="content">
+        <div class="content__image">
+            
+            <img class="content__image--1" src="./images/lady-doctor.png">
+        </div>
+        <div class="content__text">
+            <div>
+                <hr class="content__hr">
+                <label class="content__text--big">Meet our experts</label>
+                <div class="content__text--muted">
+                    Problems trying to resolve the conflict between 
+                    the two major realms of Classical physics:
+                    Newtonian mechanics 
+                    <div class="welcome__text--blue text--padded">
+                        Learn More <img src="./images/Vector.png">
+                    </div>
+                </div>
+            </div>
+        </div>
+    </div>
+    <div class="teams">
+        <div class="teams__text">
+            <label class="text--padded welcome__text--blue">
+                Team
+            </label>
+            <label class="content__text--big">Our Team</label>
+            <div class="content__text--muted">
+                Problems trying to resolve the conflict between 
+                the two major realms of Classical physics: Newtonian mechanics 
+            </div>
+        </div>
+        <div class="teams__images">
+            <div class="teams__div">
+                <div class="card">
+                    <img class="card__image" src="./images/doc1.png">
+                    <label class="card__name text--bold">
+                        Julian Jameson
+                    </label>
+                    <label class="card__proffesion text--muted">
+                        Proffesion
+                    </label>
+                    <div class="card__socials">
+                        <img src="./images/fb.png">
+                        <img src="./images/ig.png">
+                        <img src="./images/tw.png">
+                    </div>
+                </div>
+                <div class="card">
+                    <img class="card__image" src="./images/doc2.png">
+                    <label class="card__name text--bold">
+                        Julian Jameson
+                    </label>
+                    <label class="card__proffesion text--muted">
+                        Proffesion
+                    </label>
```

Comment:
If `text--bold` is a modifier specific to these elements, best practice would be `<label class="card__name card__name--text-bold"`

Reply:
📌